README.md

```diff
@@ -0,0 +1,36 @@
+# EmptyChronicle
+
+EmptyChonicle (or 0c) is simple node for NineChronicles providing http api about block, state, etc.
```

Comment:
After reading the content below, I thought it would be better to unify it.
```suggestion
EmptyChonicle (or 0c) is simple node for NineChronicles providing HTTP APIs about block, state, etc.
```